BACKEND.md

```
@@ -285,35 +296,67 @@ supabase secrets set SUPABASE_DB_URL=your-db-url
+   To run a specific function locally for testing:
+```bash
+   supabase functions serve <function-name>
+   ```
+
+   Replace <function-name> with the name of the function you want to test, for example:
+
+```bash
+   supabase functions serve fetch-transcript
+   ```
+
+The CLI will start a local server, usually on http://localhost:54321/functions/v1/<function-name>. You can make HTTP requests to this endpoint to test your function.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)